oauth2client/service_account.py

```diff
@@ -683,3 +666,44 @@ def _create_token(self, additional_claims=None):
         jwt = crypt.make_signed_jwt(self._signer, payload,
                                     key_id=self._private_key_id)
         return jwt.decode('ascii'), expiry
+
+    def apply(self, headers, access_token=None):
+        """Add the access token as the Authorization header in headers.
+
+        Args:
+            headers: dict, the headers to add the Authorization header to.
+            access_token: An access token to use instead of the credential's
+                current access token.
+        """
+        if not access_token:
+            access_token = self.access_token
+        headers['Authorization'] = 'Bearer ' + access_token
```

Comment:
At current, `httplib2`[ normalizes][1] the headers to be lowercase

[1]: https://github.com/jcgregorio/httplib2/blob/010b4e399c04a199e7ca25995286f3db0337500d/python2/httplib2/__init__.py#L263-L264